crimsobot/cogs/games.py

```diff
@@ -598,6 +598,75 @@ async def daily(self, ctx: commands.Context, lucky_number: int) -> None:
 
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=['bw'])
+    @commands.cooldown(5, 30, commands.BucketType.channel)
+    async def bubblewrap(self, ctx: commands.Context, *, bubble: str = None) -> None:
+        """Get a little sheet of bubble wrap to pop!
+        Provide any standard emoji or custom emoji from this server to pop!
+        If you use the command by itself, get a random emoji!
+        Short text inputs can also be used.
+
+        There is a per-channel cooldown on this command."""
+
+        def choose_random_emoji() -> str:
+            """Choose a random emoji from the emojis used in Emojistory."""
+            # read in lines of emojis
+            emojis = open(c.clib_path_join('games', 'emojilist.txt'), encoding='utf8', errors='ignore').read()
+            # strip newlines
+            emojis = emojis.replace('\n', '')
+            # choose and return
+            return random.choice(emojis)
+
+        # configure
+        char_limit = 10
+        size = 6
+
+        # send this if there's a whoopsie
+        error_embed = c.crimbed(
+            title='**OOPSIE!**',
+            descr='\n'.join([
+                'Alright, so either:',
+                '· that emoji is not in this server,',
+                '· after cleaning up your input, there are no characters left, or',
+                f'· your input is too long. (Under {char_limit} characters pls!)',
+                '',
+                'Some Discord objects such as mentions have hidden characters that will make the input too long.'
+            ]),
+            thumb_name='weary',
+            color_name='yellow',
+            footer='Please try again!'
+        )
```

Comment:
i like the thought behind this, but the engineer in me says that specific errors are always better...
maybe put it in a nested function and just pass in the error message where applicable?

Reply:
This should eventually get wrapped into custom crimsoBOT errors. We've talked about it before, I guess it's almost time to do that. If you're ok with it, we can leave this here for now and move it to some crimsoBOT error module in the future?

Reply:
works for me